frontend/app/login/page.jsx

```diff
@@ -11,21 +11,70 @@ const Login = () => {
 
     const signIn = async () => {
         try {
-            const { user: { uid } } = await signInWithPopup(auth, provider);
+            const {
+                user: { uid },
+            } = await signInWithPopup(auth, provider);
             Cookies.set('uid', uid);
-            router.push('/dashboard'); 
-        } catch(err) {
+            router.push('/teams');
+        } catch (err) {
             console.log(err);
         }
-    }
-    
+    };
+
     return (
         <div className={loginStyles.container}>
-            <button onClick={() => signIn()}>
-                 { `G -> Sign in with Google` }
-            </button>
+            <div className={loginStyles.blurbBackground}>
+                <div className={loginStyles.blurbContainer}>
+                    <div className={loginStyles.headerContainer}>
+                        <h1 className={loginStyles.headerAssistant}>Assistant</h1>
```

Comment:
This is an important topic: From an accessibility perspective, every page should only have a single H1 element. You can use spans here to fix this line:
```
<h1>Assistant <span className="captain">Captain</span></h1>
```